conf/conf.go

```diff
@@ -56,6 +56,7 @@ func NewConfiguration(path string) (*Configuration, error) {
 	v.AddConfigPath(".")
 	v.AddConfigPath("$HOME")
 
+	v.SetDefault("Token", "")
```

Comment:
I see that `conf` files were added to the PR by mistake I believe. Try to rebase your branch against the `main` and force push so that they would not be part of the diff